components/raftstore/src/store/util.rs

```diff
@@ -1733,13 +1733,28 @@ pub struct RaftstoreDuration {
 }
 
 impl RaftstoreDuration {
+    #[inline]
     pub fn sum(&self) -> std::time::Duration {
-        self.store_wait_duration.unwrap_or_default()
-            + self.store_process_duration.unwrap_or_default()
+        self.delays_on_disk_io(true) + self.delays_on_net_io()
+    }
+
+    #[inline]
+    pub fn delays_on_disk_io(&self, include_wait_duration: bool) -> std::time::Duration {
+        let duration = self.store_process_duration.unwrap_or_default()
             + self.store_write_duration.unwrap_or_default()
-            + self.store_commit_duration.unwrap_or_default()
-            + self.apply_wait_duration.unwrap_or_default()
-            + self.apply_process_duration.unwrap_or_default()
+            + self.apply_process_duration.unwrap_or_default();
+        if include_wait_duration {
+            duration
+                + self.store_wait_duration.unwrap_or_default()
+                + self.apply_wait_duration.unwrap_or_default()
+        } else {
+            duration
+        }
+    }
+
+    #[inline]
+    pub fn delays_on_net_io(&self) -> std::time::Duration {
```

Comment:
It's a bit misleading for using commit_duration for "net_io".
Because commit_duration can cover net io issue, but it can cover other issues as well (anything makes the raft message slowly processed in other tikv nodes could lead to large store_commit_duration)

So maybe we can just rename the fn to delays_on_commit_duration and add comments that network latency is the most common reason for it.